batch.go

```diff
@@ -16,7 +16,7 @@ type BatchWithFlusher struct {
 var _ dbm.Batch = &BatchWithFlusher{}
 
 // Ethereum has found that commit of 100KB is optimal, ref ethereum/go-ethereum#15115
-var defaultFlushThreshold = 100000
+// var defaultFlushThreshold = 100000
```

Comment:
maybe delete the dead code?

Reply:
it came from another PR, seems like using it as a reference description. just comment to avoid lint failures